content/articles/2018-08-05-google-summer-of-code-2018.md

```diff
@@ -0,0 +1,24 @@
+slug: 2018-08-05-google-summer-of-code-2018
+lang: en
+Author: Nikita Astrakhantsev
+
+It was a lazy sunday at home and nothing prepared me for my GF giving me the link for the GSoC'18 announcement page. I was considering participation a year ago, but had not enough resoluteness for that. This year I wanted it to be different so I started scrolling through the list of open-source organisations. As I am doing a PhD in theoretical physics now (defending in Jnauary 2019), I wanted the project to be scientific, not just some boring back-end development (in which I am anyway not strong enough). After a couple of iterations I converged to three space-aimed organisations. What shocked me, was the fact that mostly people of these organizations were not eager to even answer simple questions or it took them a week to provide a dataset for analysis. Only in Poliastro it was easy enough to get in touch with people and they (especially Juan) turned out to be friendly and helpful. In such an atmosphere it is more than possible to work.
+
+After looking project proposal list, I found what I am strong at and made a compilation of three project proposals into my own final [proposal](https://storage.googleapis.com/summerofcode-prod.appspot.com/gsoc/core_project/doc/5416846749073408_1521547584_proposal_astrakhantsev.pdf?Expires=1533564319&GoogleAccessId=summerofcode-prod%40appspot.gserviceaccount.com&Signature=OkUaCTO50Xy4dLVGKpUPlbsqj2XYI3QZRX8GriDapFoSpjV0xs%2F4zy2WD%2Fwj1dNZHxHYXppE7GR7i6D0cDjuarlWp6OwYPnoiBdCfh7Fg3UG8WjGWjOXE9oP8Jer2QA3teG5U9xYHgtStzTd9uQRA%2FA14qOQg0nVzJvCocIjYVWPROipz9V08uzhMbPISEX4AlGs2G5KC4F08P3D%2Bf4AaYsh6euug6JxY40zOrOZCwm%2Bg0Jh6s1kpB9XDeIzMb9Qdb06YFpLpQUggVNW7v7fPxuM188qEiNTdMR%2BvuMhnpgC1cV22%2FIB3l3xvPlFiJ%2BjpoDIw%2FX1JXHbfeQpl%2BUQzg%3D%3D) and got accepted.
+
+During the GSoC, I was writing a [blog on Medium](https://medium.com/@nikita.astronaut). You can actually see how I was becoming more and more tired till the end as the length of post was gradually shrinking :) Still, there I was trying to describe all the things we've done and give all the links to merged pull requests. Below I will repeat that in more of a shorter manner, giving links to PRs and corresponding Medium posts :)
+
+## So what was done (in chronological order)?
+1. [Bonding period](https://medium.com/@nikita.astronaut/my-bonding-period-working-for-gsoc18-poliastro-project-3aa10732b7c0). Fixed non-smoothness in orbit-plotting [here](https://github.com/poliastro/poliastro/pull/320), added mean_motion propagator [here](https://github.com/poliastro/poliastro/pull/322) and made it possible to propagate orbit to desired true anomaly [here](https://github.com/poliastro/poliastro/pull/334). Added J2 perturbation [here](https://github.com/poliastro/poliastro/pull/341), atmospheric drag [here](https://github.com/poliastro/poliastro/pull/343), fixed Newton method and Kepler propagator convergence [here](https://github.com/poliastro/poliastro/pull/362).
```

Comment:
Can you use backticks for all the code references? `mean_motion`, `Orbit.sample` (also: ~~orbit~~ `Orbit`), etc

Reply:
Also, perhaps you want to replace things like

> Fixed non-smoothness in orbit-plotting [here](https://github.com/poliastro/poliastro/pull/320), added mean_motion propagator [here](https://github.com/poliastro/poliastro/pull/322)

by

> [Fixed non-smoothness in orbit-plotting](https://github.com/poliastro/poliastro/pull/320), [added mean_motion propagator](https://github.com/poliastro/poliastro/pull/322)